CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Contribution Guidelines
+
+### Contributor Agreement
```

Comment:
I don't think CLA applies here. You can remove this section.

Reply:
Removed.